packages/components/src/color-palette/README.md

```diff
@@ -1,21 +1,31 @@
 # ColorPalette
 
+ColorPalette allows the user to pick a color from a list of pre-defined color entries.
+
 ## Props
 
 The component accepts the following props.
 
-{ colors, disableCustomColors = false, value, onChange, className, clearable = true }
+{ colors = [],  disableCustomColors = false, enableAlpha, value, onchange, className, clearable = true, __experimentalHasMultipleOrigins = false, __experimentalIsRenderedInSidebar = false }
 
 ### colors
-
-Array with the colors to be shown.
+Array with the colors to be shown. When displaying multiple color palettes to choose from, the format of the array changes from an array of colors objects, to an array of color palettes.
 
 -   Type: `Array`
--   Required: Yes
+-   Required: No
+-   Default: `[]`
```

Comment:
A bit nitty, but let's try to follow the format highlighted in the [guidelines](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/CONTRIBUTING.md#readme-example):

```markdown
### `propName`: Typescript style type i.e `string`, `number`, `( nextValue: string ) => void`

Prop description. With a new line before and after the description and before and after type/required blocks.

-   Required: Either `Yes` or `No`
<!-- If the prop has a default value, add the following line: -->
-   Default: [default value]
```

For example, for the `color` prop the docs would change to

```markdown
### `colors`: `( PaletteObject | ColorObject )[]`

Array with the colors to be shown. When displaying multiple color palettes to choose from, the format of the array changes from an array of colors objects, to an array of color palettes.

-   Required: No
-   Default: `[]`
```

Reply:
Oh, good idea. Sorry for missing that in CONTRIBUTING.md. 

Committed in https://github.com/WordPress/gutenberg/pull/44632/commits/988e9401da364236c2dfe0296fe4cd234029ae4b